.github/workflows/ci_ckb_cli.yaml

```diff
@@ -0,0 +1,64 @@
+name: ci_ckb_cli
+on:
+  # Submit your review with a comment body containing "#ci_ckb_cli"
+  pull_request_review:
+    types: [ submitted ]
+  push:
+    branches:
+      - 'pkg/*'
+  workflow_dispatch:
+
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: full
+  RUSTFLAGS: -D warnings
+jobs:
+  ckb_cli:
+    runs-on: ${{ self-hosted, linux }}
```

Comment:
`${{ }}` is a syntax to use a GitHub Action variable, here `self-hosted, linux` is not a valid variable name.

Please refer to `ci_integration_tests_ubuntu.yaml` how to set up the runner using a prologue script.

Reply:
done